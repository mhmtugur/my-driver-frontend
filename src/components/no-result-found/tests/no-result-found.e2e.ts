import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';

describe('no-result-found', () => {
  let page: E2EPage;
  let element: E2EElement;
  beforeEach(async () => {
    page = await newE2EPage({ html: '<no-result-found></no-result-found>' });
    element = await page.find('no-result-found');
  });

  it('should render no result found message', async () =>{
   expect(element.innerHTML).toMatchSnapshot();
  })
});