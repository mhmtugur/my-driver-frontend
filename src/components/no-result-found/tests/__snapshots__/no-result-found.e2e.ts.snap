// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-result-found should render no result found message 1`] = `"<div class=\\"result\\"><h5>No Result Found!</h5></div>"`;
